server/app/views/v1/host_nodes/health.json.jbuilder

```diff
@@ -0,0 +1,3 @@
+json.id @node.node_id
+json.name @node.name
+json.etcd @etcd_health
```

Comment:
The reason this is a separate JSON key is that this response would be extended with other health signals in the future, like `{"weave": ..., "etcd": ..., "ipam": ...}` etc.